bin/push-debian.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Example usage:
+#
+# bin/push-debian.sh \
+#   -c opt
+#   -v 0.2.1
+
+function usage() {
+  echo "$0 \
+    -c <bazel config to use> \
+    -v <istio version number>"
+  exit 1
+}
+
+while getopts ":c:v:" arg; do
+  case ${arg} in
+    c) BAZEL_ARGS="--config=${OPTARG}";;
+    v) ISTIO_VERSION="${OPTARG}";;
+    *) usage;;
+  esac
+done
+
+if [ -z "${BAZEL_ARGS}" ] || [ -z "${ISTIO_VERSION}" ]; then
+  usage
+fi
+
+set -ex
+
+bazel ${BAZEL_STARTUP_ARGS} build ${BAZEL_ARGS} "//tools/deb:istio-auth-node-agent"
+gsutil -m cp -r \
+  bazel-bin/tools/deb/istio-auth-node-agent_${ISTIO_VERSION}_amd64.* \
+  gs://istio-release/releases/${ISTIO_VERSION}/deb
```

Comment:
Could you please make configurable like in here:
https://github.com/istio/pilot/blob/master/bin/upload-istioctl#L23

Reply:
How do you want this to work?  In upload-istioctl you pass it the full GCS path, however for the debian package the version number is included in the path.  We can do the path prefix, so something like:
push-debian.sh -p gs://istio-release/releases
gsutil -m cp -r *.deb ${BUCKET_PATH}/${ISTIO_VERSION}/deb

I don't really see this as an improvement.  I could see doing something like:
push-debian.sh -b istio-release
gsutil -m cp -r *.deb gs://${BUCKET_NAME}/releases/${ISTIO_VERSION}/deb

Alternatively we could just move all the logic to the release script with something like:
push-debian.sh -b gs://istio-release/releases/0.2.1/deb

Reply:
Resolved.